### DISCUSSION TITLE
Link Notes

### DISCUSSION THREAD

**Discussion by hnasheralneam** [davidmyersdev/octo#220] [Ideas]
I’m not sure what the idea is with the force graph, but if it’s anything like Obsidian’s, you need some way to connect documents. Where you planning on connecting them using tags, or also linking documents?

**Comment by davidmyersdev** [comment]
The force graph currently uses tags to connect docs, but I have plans to use direct links as well. I recently added support for the direct link syntax (e.g. `[[some doc]]`), but the UX around it still needs a lot of work.
